### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ChinYikMing**
Add a section dicussing about the benchmark CI.

#213

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +198,22 @@ _Example Instructions Histogram_
...
+There are three files which have the potential to significantly impact the performance of `rv32emu`:
+* `src/decode.c`
+* `src/rv32_template.c`
+* `src/emulate.c`
+
+Therefore, updates to these three files will start the benchmark CI. Additionally, the committer and reviewer can assess and discuss the performance of the benchmark by looking at the benchmark comparison comment between the new PR commit and the previous commit before merging the PR to the master branch.
+
+The results of the benchmark will be displayed on a github webpage: [link](https://sysprog21.github.io/rv32emu-bench/)
+
+_Example Dhrystone Benchmark Visualization_
```
Comment (jserv) [inline README.md]: Remove the example screenshots since they are almost blank now.

File: README.md [hunk 2]
```diff
@@ -198,6 +198,22 @@ _Example Instructions Histogram_
 _Example Registers Histogram_
 ![Registers Hisrogram Example](docs/histogram-reg.png)
 
+## Benchmarking
+There are three files which have the potential to significantly impact the performance of `rv32emu`:
+* `src/decode.c`
+* `src/rv32_template.c`
+* `src/emulate.c`
+
+Therefore, updates to these three files will start the benchmark CI. Additionally, the committer and reviewer can assess and discuss the performance of the benchmark by looking at the benchmark comparison comment between the new PR commit and the previous commit before merging the PR to the master branch.
```
Comment (jserv) [inline README.md]: Improve the writing. Use Grammarly or similar tools to rephrase.

Comment (jserv) [reply README.md]: Always mention pull request(s) instead of "PR."

File: README.md [hunk 3]
```diff
@@ -198,6 +198,16 @@ _Example Instructions Histogram_
...
 
+## Benchmarking
+There are three files which have the potential to significantly impact the performance of `rv32emu`:
+* `src/decode.c`
+* `src/rv32_template.c`
+* `src/emulate.c`
+
+Therefore, any modification on these three files will trigger the benchmark CI.The committer and reviewer can look at the comment of benchmark comparison between the commit of the pull request(s) and the latest commit of the master branch in the conversation and debate about it before merging. 
+
+The results of the benchmark will be displayed on a github webpage: [link](https://sysprog21.github.io/rv32emu-bench/)
```
Comment (jserv) [inline README.md]: Rewrite as following:
```
The results of the benchmark will be rendered on a [GitHub page](https://sysprog21.github.io/rv32emu-bench/).
```

File: README.md [hunk 4]
```diff
@@ -198,6 +198,16 @@ _Example Instructions Histogram_
 _Example Registers Histogram_
 ![Registers Hisrogram Example](docs/histogram-reg.png)
 
+## Benchmarking
+There are three files which have the potential to significantly impact the performance of `rv32emu`:
+* `src/decode.c`
+* `src/rv32_template.c`
+* `src/emulate.c`
+
+Therefore, any modification on these three files will trigger the benchmark CI.The committer and reviewer can look at the comment of benchmark comparison between the commit of the pull request(s) and the latest commit of the master branch in the conversation and debate about it before merging.
```
Comment (jserv) [inline README.md]: Add appropriate space right after `.`

File: README.md (hunk 3 above)
Comment (jserv) [reply README.md]: You shall mention the framework or facilities to perform benchmarks.

File: README.md (hunk 4 above)
Comment (ChinYikMing) [reply README.md]: I split it with a new line

**Pull request merged**